content/docs/other-resources.md

```diff
@@ -3,30 +3,34 @@ title: Other Resources
 weight: 160
 ---
 
-
 ## Notepad Replacement
+
 Notepad is a default text editor shipped with Windows. You may want to use Notepad++ instead of Notepad. However, there's no obvious way to do it.
-From the version 7.5.9 onward, you can run the following command to make Notepad++ replace Notepad (run in `cmd.exe` with Administrator privileges)[†](#registry-edit-warning):
+From the version 7.5.9 onward, you can run the following command to make Notepad++ replace Notepad (run in `cmd.exe` with Administrator privileges[†](#registry-edit-warning)):
 
 ```batch
 reg add "HKLM\Software\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\notepad.exe" /v "Debugger" /t REG_SZ /d "\"%ProgramFiles%\Notepad++\notepad++.exe\" -notepadStyleCmdline -z" /f
 ```
 
 Note that you may need to use `%ProgramFiles(x86)%\Notepad++\` to substitute for `%ProgramFiles%\Notepad++\` if you have Notepad++ 32-bit installed, or use other path if your Notepad++ is installed in a non-default location.
 
-Use the the following comment to undo the replacement:
+In Windows 11 there is also UWP version of Notepad installed and configured to be used instead of the legacy built-in one. To make above replacement respected, UWP Notepad must be both uninstalled (just right-click on the Notepad shortcut in **Start Menu** and select **Uninstall**) and the configuration disabled (run in `cmd.exe` with Administrator privileges[†](#registry-edit-warning)):
+
 ```batch
-reg delete "HKLM\Software\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\notepad.exe" /v "Debugger" /f
+reg add "HKLM\Software\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\notepad.exe" /v "UseFilter" /t REG_DWORD /d 0 /f
 ```
 
-This has historically worked from Windows 7 through Windows 10.  However, in Windows 11 it's Notepad UWP version installed, you have to remove Notepad UWP to make the above instructions work.
+Use the the following command to undo the replacement:
 
-To remove UWP version of Notepad, you have to launch powershell with admin privileges, then run the following command:
-```powershell
-Get-AppxPackage *Microsoft.WindowsNotepad* | Remove-AppxPackage
+```batch
+reg delete "HKLM\Software\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\notepad.exe" /v "Debugger" /f
 ```
 
-For restoring Notepad, launch "Microsoft Store" via Start menu of Windows, search "Notepad", select "Windows Notepad", then click on "Install" button.
+Additionally on Windows 11 reinstall Notepad (launch "Microsoft Store" via Start menu of Windows, search "Notepad", select "Windows Notepad", then click on "Install" button) and run:
```

Comment:
Please rephrase.  I suggest something like:

> On Windows 11, you must additionally reinstall Notepad (launch "Microsoft Store" via Start menu of Windows, search "Notepad", select "Windows Notepad", then click on "Install" button) and then run:

Reply:
Thanks for styling comments, appreciate that!

In the end I changed the instructions again, as I finally understood what exactly is done there in registry for UWP Notepad: UWP Notepad use `UseFilter` in Image File Execution Options, which allows you to match executables using full paths (in conjunction with `FilterFullPath` in subkey), which avoids the (low) risk of replacing some unrelated binary with notepad.exe as name